test/Libraries/DynamoPythonTests/DynamoPythonTests.csproj

```diff
@@ -76,6 +76,7 @@
       <Private>False</Private>
     </Reference>
     <Reference Include="PresentationCore" />
+    <Reference Include="Python.Runtime, Version=2.5.1.0, Culture=neutral, PublicKeyToken=null" />
```

Comment:
this is required because the evaluationHandlers require a reference to `pyScope` a type from pythonNet's Python.Runtime. 😢